test/suggestion.cpp

```diff
@@ -87,6 +87,12 @@ namespace {
     return result;
   }
 
+#define EXPECT_SUGGESTION_RESULTS(archive, query, ...)          \
```

Comment:
What do you think about using this macro in the other `Suggestion` unit-tests (but of course not in this PR)?

Reply:
To be honest it's pretty elegant :smile: A product of experience I guess. I will keep updating the rest of the tests whenever I get the chance with this macro.